File: AppGenerateFiles/dbo/Model/Detail_Prendas_Vehiculos.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Detail_Prendas_Vehiculos : EntityClass {
       [PrimaryKey(Identity = false)]
       public int? numero_prenda { get; set; }
       public string? capacidad_cilindros { get; set; }
       public string? cantidad_cilindros { get; set; }
       public string? cantidad_pasajeros { get; set; }
       public int? year_vehiculo { get; set; }
       public string? montor { get; set; }
       public string? chasis { get; set; }
       public string? placa { get; set; }
       public string? circuacion { get; set; }
       public string? defectuoso { get; set; }
       public DateTime? fecha_aut_descuento { get; set; }
       public string? defecto { get; set; }
       public Double? porcentage_descuento_maximo { get; set; }
       public DateTime? fecha_seguro { get; set; }
       public string? combustible { get; set; }
       public string? uso { get; set; }
       public string? servicio { get; set; }
       [OneToOne(TableName = "Detail_Prendas", KeyColumn = "numero_prenda", ForeignKeyColumn = "numero_prenda")]
       public Detail_Prendas? Detail_Prendas { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Detail_Prendas_Vehiculos.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Detail_Prendas }  from './Detail_Prendas.js'
class Detail_Prendas_Vehiculos extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {String}*/ capacidad_cilindros;
   /**@type {String}*/ cantidad_cilindros;
   /**@type {String}*/ cantidad_pasajeros;
   /**@type {Number}*/ year_vehiculo;
   /**@type {String}*/ montor;
   /**@type {String}*/ chasis;
   /**@type {String}*/ placa;
   /**@type {String}*/ circuacion;
   /**@type {String}*/ defectuoso;
   /**@type {Date}*/ fecha_aut_descuento;
   /**@type {String}*/ defecto;
   /**@type {Number}*/ porcentage_descuento_maximo;
   /**@type {Date}*/ fecha_seguro;
   /**@type {String}*/ combustible;
   /**@type {String}*/ uso;
   /**@type {String}*/ servicio;
   /**@type {Detail_Prendas} ManyToOne*/ Detail_Prendas;
}
export { Detail_Prendas_Vehiculos }
